common/errors.js

```diff
@@ -180,7 +180,7 @@
 
             var params = {
                 message: message,
-                errorCode: errorCode,
+                errorStatus: errorStatus,
```

Comment:
`errorCode` has been renamed to `errorStatus` as `errorCode` (the actual error http like code) is introduced; also erstwhile variable name (`errorCode`)  was confusing since it was holding a string instead of any code.